src/components/schemas/Average.yaml

```
@@ -0,0 +1,12 @@
+type: object
+properties:
+  count:
+    type: integer
+    format: int64
+    example: 7568
```

:sparkles: suggestion: Ajouter la contrainte de positivité :
```suggestion
    example: 7568
    minimum: 0
```